build.gradle

```diff
@@ -88,6 +88,16 @@ task travisRelease {
     }
 }
 
+task installGitPreCommitHook(type: Copy) {
+    from 'pre-commit'
+    into '.git/hooks'
+}
+
+task makeGitHooksExecutable { Task t ->
```

Comment:
- [ ] - I suggest a simple name like "installPrecommit". Easier to use :)

- [ ] - Please add descriptions to the tasks and group "precommit", they are visible when one runs "gradle tasks" and make the project easier to work with.